src/main/kotlin/org/rust/ide/inspections/checkMatch/RsUselessMatchArmInspection.kt

```diff
@@ -0,0 +1,79 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.inspections.checkMatch
+
+import com.intellij.openapi.util.TextRange
+import com.intellij.psi.PsiElement
+import com.intellij.psi.impl.source.tree.LeafPsiElement
+import org.rust.ide.inspections.RsProblemsHolder
+import org.rust.ide.inspections.fixes.SubstituteTextFix
+import org.rust.ide.inspections.lints.RsLint
+import org.rust.ide.inspections.lints.RsLintInspection
+import org.rust.lang.core.psi.RsElementTypes.OR
+import org.rust.lang.core.psi.RsMatchArm
+import org.rust.lang.core.psi.RsMatchExpr
+import org.rust.lang.core.psi.RsVisitor
+import org.rust.lang.core.psi.ext.*
+import org.rust.lang.core.types.infer.containsTyOfClass
+import org.rust.lang.core.types.ty.TyUnknown
+import org.rust.lang.core.types.type
+
+class RsUselessMatchArmInspection : RsLintInspection() {
```

Comment:
```suggestion
class RsUselessMatchPatternInspection : RsLintInspection() {
```

Reply:
Let's use `RsUnreachablePatternsInspection`? It is located in `checkMatch` package so it's easy to discover that it is only about match expressions

Reply:
👍